src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1319,5 +1319,6 @@ internal enum ErrorCode
         ERR_BadAwaitInStaticVariableInitializer = 8100,
         ERR_InvalidPathMap = 8101,
         ERR_PublicSignButNoKey = 8102,
+        ERR_TooManyUserStrings = 8103,
```

Comment:
Heads up: this conflicts with the value for ERR_ExpressionTreeContainsLocalFunction in future.